### DISCUSSION TITLE
Update documentation

### DISCUSSION THREAD

**Discussion by EvgenyGulyaev** [bcakmakoglu/vue-flow#630] [Q&A]
Can you add in the readme that vue-cli is supported up to version 1.12.0 ? Because I get the following error
Module parse failed: Unexpected token (96:82)

You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
```
|     if (!cr(d))
|       return n((w) => w === d, h);
>     const { flush: f = "sync", deep: x = !1, timeout: v, throwOnTimeout: _ } = h ?? {};
|     let m = null;
|     const k = [new Promise((w) => {
```

**Comment by bcakmakoglu** [comment]
I'll check out what's happening, thanks :)

But -
Is there a specific reason you use vue-cli instead of the recommended way and scaffolding a vite project?
<img width="784" alt="image" src="https://user-images.githubusercontent.com/78412429/214838927-5815e596-ec7c-402c-ab53-da3a17c50237.png">

**Reply by EvgenyGulyaev** [reply]
The project is too big, dependencies too. Well, we just can't allocate time for this because of the large count of tasks in the project

**Reply by bcakmakoglu** [reply]
I see, still would be a good idea to put it on the list of things to do as vue-cli is, as the picture shows, not really the way to go anymore (aside from the fact that webpack is horrible at this point 😅)
I'll still look into the problem and see if I can resolve it, sorry for the inconvenience
